internal/server/spanner/query.go

```diff
@@ -58,17 +60,29 @@ var statements = struct {
 	FROM
 		Edge edge
 	LEFT JOIN
-		graph_table( DCGraph match -[e:Edge
+		GRAPH_TABLE( DCGraph MATCH -[e:Edge
 		WHERE
 			e.subject_id IN UNNEST(@ids)
-			AND e.object_value IS NULL]->(n:Node) return n.subject_id,
+			AND e.object_value IS NULL]->(n:Node) RETURN n.subject_id,
 			n.name,
 			n.types) object
 	ON
 		edge.object_id = object.subject_id
 	WHERE
 		edge.subject_id IN UNNEST(@ids)
 	`,
+	getEdgesByObjectID: `
+	GRAPH DCGraph MATCH (n:Node)-[e:Edge
+	WHERE
+		e.object_id IN UNNEST(@ids)
```

Comment:
Have we created an index by object_id? Any difference in performance between the 2 queries?